src/Compilers/CSharp/Test/Semantic/Semantics/UsingStatementTests.cs

```diff
@@ -1452,6 +1453,280 @@ static void Main()
             CreateCompilation(source).VerifyDiagnostics();
         }
 
+        [Fact]
+        public void UsingPatternAsyncTest()
+        {
+            var source = @"
+namespace System
+{
+    public interface IAsyncDisposable
+    {
+        System.Threading.Tasks.ValueTask DisposeAsync();
+    }
+}
+class C1
+{
+    public C1() { }
+    public System.Threading.Tasks.ValueTask DisposeAsync() { System.Console.WriteLine(""Dispose async""); return new System.Threading.Tasks.ValueTask(System.Threading.Tasks.Task.CompletedTask); }
```

Comment:
>{ [](start = 59, length = 1)

Please split the method body into multiple lines, or add `using System.Threading.Tasks;`, or both.